resources/providers/awslib/ec2_metadata_getter_mock.go

```diff
@@ -0,0 +1,103 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Code generated by mockery v2.14.0. DO NOT EDIT.
+
+package awslib
+
+import (
+	context "context"
+
+	aws "github.com/aws/aws-sdk-go-v2/aws"
+
+	ec2metadata "github.com/aws/aws-sdk-go-v2/aws/ec2metadata"
+
+	mock "github.com/stretchr/testify/mock"
```

Comment:
redundant import naming?

Reply:
It's part of the auto-generated file - but I'll change

Reply:
ignore my comments regarding the auto-generated file